manage_breast_screening/clinics/views.py

```diff
@@ -28,7 +28,9 @@ def clinic_list(request, filter="today"):
 def clinic(request, id, filter="remaining"):
     clinic = Clinic.objects.prefetch_related("setting").get(id=id)
     presented_clinic = ClinicPresenter(clinic)
-    appointments = Appointment.clinic_appointments_by_filter(clinic, filter)
+    appointments = Appointment.clinic_appointments_by_filter(
+        clinic, filter
+    ).select_related("clinic_slot", "screening_episode__participant")
```

Comment:
Thanks for adding this. I've also been wondering if we have access to any application monitoring tools like Sentry that would automatically flag these kinds of issues for us. @gpeng  do you know?